### PR TITLE
feat: add playerctl module to waybar

### LINKED ISSUE #124: Widget for Waybar, allowing you to manage multimedia.
Widget for Waybar, allowing you to manage multimedia.
In most WM, this feature is implemented by means of notifications.
It is based on MPRIS D-BUS Interface, is not attached to a specific player. This is implemented through the [playerctl](https://github.com/altdesktop/playerctl) program.

Scenario of use: The music is played in the background, in my case in Firefox. Suddenly there is a call to a cell phone. I need to respond urgently. Looking for a tab with audio players usually take a long time. This module will allow you to do in one click. As well as will allow the playback as quickly.

The widget tried to make the most compact. If you completely turn off the audio player, the widget will automatically hide from the panel.

`sway/etc/skel/.config/waybar/config`
~~~json
  "custom/playerctl": {
	"interval": 1,
	"tooltip": true,
	"return-type": "json",
	"format" : "{icon}",
	"format-icons": {
		"Playing": "喇",
		"Paused": ""
	},
	"exec": "status=$(playerctl status) && title=$(playerctl metadata title) && artist=$(playerctl metadata artist) && echo {'\"alt\"':'\"'$status'\"', '\"tooltip\"':'\"'$title - $artist'\"'}",
	"on-click": "playerctl play-pause",
	"on-click-right": "playerctl next",
	"on-scroll-up": "playerctl position 10+", 
	"on-scroll-down": "playerctl position 10-", 
  },
~~~
`sway/etc/skel/.config/waybar/config`
~~~json
"modules-right": [
....
	"custom/playerctl",
....
~~~
`sway/etc/skel/.config/waybar/style.css`
~~~css
/* Each module */
....
#custom-playerctl,
....
~~~

### PR DESCRIPTION
**Pull request by boredland**
closes https://github.com/Manjaro-Sway/manjaro-sway/issues/124

### CODE REVIEW

**Comment by FallenChromium** [comment]
"interval: 1" sounds like a battery hog to me. Any way to implement it with an interrupt signal?

**Comment by boredland** [comment]
I think its reasonable to increase the inverval, but as for emitting events, I guess the main problem is, that we don't know the emitter...

**Comment by boredland** [comment]
perhaps we can just add sth like this as an autostart?

```
playerctl metadata --follow | (pkill -RTMIN+8 waybar)
```

**Comment by FallenChromium** [comment]
Exactly, found the same thing on [Reddit](https://www.reddit.com/r/linuxquestions/comments/k8cw8m/playerctl_signals/gexeblm?utm_source=share&utm_medium=web2x&context=3) :)

**Comment by FallenChromium** [comment]
By the way, using the same signal to update all the modules is probably a bad idea. We have used signal 8 for wf-recorder, so it will be good to use some other number for playerctl. Just not to reload unnecessary modules

File: community/sway/etc/sway/config.d/99-autostart-applications.conf [hunk 1]
```diff
@@ -18,4 +18,5 @@ exec_always {
     # restart kanshi https://github.com/emersion/kanshi/issues/43#issuecomment-531679213
     '[ -x "$(command -v kanshi)" ] && pkill kanshi; exec kanshi'
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
+    "pkill -0 playerctl; playerctl --follow metadata | pkill -RTMIN+5 waybar"
```
Comment (FallenChromium) [inline community/sway/etc/sway/config.d/99-autostart-applications.conf]:
```suggestion
'[-x "$(command -v playerctl)" ] && pkill -0 playerctl; playerctl --follow metadata | pkill -RTMIN+5 waybar' 
```
Just to be more standard and stuff :D

**Review by FallenChromium — APPROVED** [review]
Looks good to me! If it works on your machine I'd be glad to get this player in a `manjaro-sway-settings-git` update after this PR gets merged :D

File: community/sway/etc/sway/config.d/99-autostart-applications.conf [hunk 2]
```diff
@@ -18,4 +18,5 @@ exec_always {
     # restart kanshi https://github.com/emersion/kanshi/issues/43#issuecomment-531679213
     '[ -x "$(command -v kanshi)" ] && pkill kanshi; exec kanshi'
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
+    '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format '' --follow | while read line; do pkill -RTMIN+5 waybar; done'
```
Comment (FallenChromium) [inline community/sway/etc/sway/config.d/99-autostart-applications.conf]:
```suggestion
    '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format '{{status}} {{title}}' --follow | while read line; do pkill -RTMIN+5 waybar; done'
```
Without those it won't trigger a newline, and thus won't trigger a reload

**Pull request merged**